spec/system/available_kudos_spec.rb

```diff
@@ -48,5 +53,6 @@ def create_new_kudo
   click_link 'New Kudo'
   fill_in 'Title', with: 'Great Worker!!'
   fill_in 'Content', with: 'Three times faster than others!'
+  select company_value.title, from: 'Company value'
```

Comment:
helper methods in specs are not really used - consider using `before` block or something offered by rspec for cases like this